Packs/ApiModules/Scripts/IAMModule/README.md

```diff
@@ -0,0 +1,14 @@
+To use the common IAM API logic, run the following command to import the `IAMModule`.
```

Comment:
let's include here information about the classes in IAMModule (use the docstrings if you aren't sure what is the purpose of each one)